php-practice_1.php

```diff
@@ -0,0 +1,143 @@
+<?php
+// Q1 変数と文字列
+$name = "福島";
+echo "私の名前は「" . $name . "」です。";
+
+// Q2 四則演算
+$num = 5 * 4;
+echo $num  . "\n";
+
+$result = $num / 2;
+echo $result   . "\n";
+
+// Q3 日付操作
+$time = date("Y年m月d日 H時i分s秒");
+
+
+// Q4 条件分岐-1 if文
+$device='windows';
+
+if ($device == 'windows') {
+  echo '使用OSは、windowsです。';
+} elseif ($device == 'mac') {
+    echo '使用OSは、macです。';
+} else {
+    echo 'どちらでもありません。';
+}
+
+// Q5 条件分岐-2 三項演算子
+$age = 10;
+$message = ($age >= 18) ? '成人です。' : '未成年です。';
+
+echo $message;
+
+// Q6 配列
+$japan = ['東京都','埼玉県','栃木県','千葉県','神奈川県','茨城県','群馬'];
+$kanto3 = $japan[3];
+$kanto4 = $japan[4];
+echo $kanto3 . "と" . $kanto4 . "は関東地方の都道府県です。";
```

Comment:
[info]
変数に代入せずとも実行可能です！
```suggestion
echo $japan[3] . "と" . $japan[4] . "は関東地方の都道府県です。";
```